src/common.props

```diff
@@ -17,6 +17,7 @@
     <ProtobufVersion>3.14.0</ProtobufVersion>
     <NetCoreTestVersion>netcoreapp3.1</NetCoreTestVersion>
     <NetFrameworkTestVersion>net461</NetFrameworkTestVersion>
+    <NetTestVersion>net5.0</NetTestVersion>
```

Comment:
I added this so that it can be added to `TargetFrameworks(<TargetFrameworks>$(NetFrameworkTestVersion);$(NetCoreTestVersion);$(NetTestVersion)</TargetFrameworks>)` for all Test projects - the non-test projects only had `TargetFrameworks` set to `NetStandardLibVersion`

Reply:
Just change the `NetCoreTestVersion` to `net5.0` - _but I would strongly recommend decoupling changing the build system from the test suite_. Do it in two separate PRs

Reply:
ah, I get why you're doing this now - have to support two different versions of the SDK to do it side by side which can get messy. Very well - go ahead and change both at the same time.